.github/workflows/ci.yaml

```diff
@@ -364,15 +364,15 @@ jobs:
         run: |
           pip install pyyaml
           pip install ccm
-
+  
```

Comment:
nit:
would be nice to not have these single space character additions as changes.

Reply:
definitely, sorry about that